packages/sdk-core/tests/_mocks/fileSystem.ts

```diff
@@ -2,11 +2,11 @@ import path from 'path';
 import { BacktraceFileAttachment } from '../../src/model/attachment';
 import { FileSystem } from '../../src/modules/storage/FileSystem';
 
-type MockedFileSystem = {
-    [K in keyof FileSystem]: jest.Mock<ReturnType<FileSystem[K]>, Parameters<FileSystem[K]>>;
+export type MockedFileSystem<T extends FileSystem> = {
```

Comment:
can we fix the warning and add es-lint ignore?

Reply:
Added.